Add tag overrides as hoot export option

rmrice opened the pull request:
`http://localhost:8101/export/API/OSM/Shapefile?bbox=-77.44911,37.53927,-77.44587,37.54333&overrideTags=true` will return:

```
wget -q  -O /home/vagrant/hoot/node-export-server/export_6e1e9550-f26b-11e7-aef6-0da8a3f2c6dd.osm http://192.168.33.12:3000/api/0.6/map?bbox=-77.44911,37.53927,-77.44587,37.54333 && hoot osm2ogr -D writer.include.debug.tags=true -D tags.override={"attribution":"","security:resource_owner":"","security:classification":"","security:dissemination:control:ic":"","security:dissemination:control:non_ic":"","security:releasability":"","source:":"","source:non_spatial_source:type":"","source:copywrite":""} translations/RenderDb.js "/home/vagrant/hoot/node-export-server/export_6e1e9550-f26b-11e7-aef6-0da8a3f2c6dd.osm" /home/vagrant/hoot/node-export-server/export_6e1e9550-f26b-11e7-aef6-0da8a3f2c6dd.shp
```

leaving the `overrideTags` part off (`http://localhost:8101/export/API/OSM/Shapefile?bbox=-77.44911,37.53927,-77.44587,37.54333`) will return:

```
wget -q  -O /home/vagrant/hoot/node-export-server/export_821869f0-f26b-11e7-aef6-0da8a3f2c6dd.osm http://192.168.33.12:3000/api/0.6/map?bbox=-77.44911,37.53927,-77.44587,37.54333 && hoot osm2ogr -D writer.include.debug.tags=true translations/RenderDb.js "/home/vagrant/hoot/node-export-server/export_821869f0-f26b-11e7-aef6-0da8a3f2c6dd.osm" /home/vagrant/hoot/node-export-server/export_821869f0-f26b-11e7-aef6-0da8a3f2c6dd.shp
```

rmrice commented:
@brianhatchl no updates to the UI for mapedit-osm yet, but that should be very straightforward.

brianhatchl commented on node-export-server/server.js:
```diff
@@ -256,12 +259,14 @@ function doExport(req, res, hash, input) {
         if (isFile) {
             command += ' convert';
             if (bbox) command += ' -D ' + bbox_param + '=' + bbox;
+            if (req.query.overrideTags) command += ' -D tags.override=' + overrideTags;
```
Looks like the param is `translation.override`

brianhatchl replied:
And the overrideTags json definitely need to be escaped.  I'm getting `Error running convert: convert takes two parameters.` with this command logged:

>hoot convert -D convert.bounding.box=-72.49812,18.54086,-72.49374,18.54270 -D translation.override={"attribution":"","security:resource_owner":"","security:classification":"","security:dissemination:control:ic":"","security:dissemination:control:non_ic":"","security:releasability":"","source:":"","source:non_spatial_source:type":"","source:copywrite":""} "osmapidb://vagrant:vagrant!@192.168.33.12:5432/openstreetmap" /home/vagrant/hoot/node-export-server/export_63edf3a0-f496-11e7-b1ae-2f5428121067.osm

brianhatchl commented on node-export-server/server.js:
```diff
@@ -31,7 +31,8 @@ app.get('/options', function(req, res) {
         Schema: _.keys(config.schemas),
-        Format: _.keys(config.formats)
+        Format: _.keys(config.formats),
+        SetTagOverrides: JSON.stringify(config.tagOverrides)
```
Putting this in the options result has this undesired effect without further UI changes.  (Defaults to a drop down, but doesn't know how to handle the stringified json.  Other options just present an array used in the dropdown).
![screenshot from 2018-01-08 11-52-18](https://user-images.githubusercontent.com/2600932/34682233-4361e832-f46c-11e7-9bc5-ae562192bb1c.png)

rmrice commented:
I linked the UI branch above that this goes with this PR.

- fixed the hoot command so that it is `translation.overrides`
- removed the list of tags from /options since we don't need it there
- escaped json

brianhatchl commented:
Let's add support to accept a json string for tagOverrides.
